src/jira/util/jwt.ts

```diff
@@ -184,6 +184,19 @@ export const verifySymmetricJwtTokenMiddleware = (secret: string, tokenType: Tok
 
 const ALLOWED_BASE_URLS = [BASE_URL]
 
+const isStagingTenant = (req: Request): boolean => {
+	let isStagingTenant = false;
+	const hostBaseUrl = _.get(req.body, "baseUrl");
```

Comment:
why are you using `_.get` just to get a property of body?

Reply:
I copied this code from ACE and left it almost unchanged. I don't think we should change much there to make sure that it will work in the same way as ACE.